replications/service.go

```diff
@@ -317,6 +319,12 @@ func (s *service) WritePoints(ctx context.Context, orgID platform.ID, bucketID p
 		return s.localWriter.WritePoints(ctx, orgID, bucketID, points)
 	}
 
+	if s.instanceID != "" {
+		for i := range points {
+			points[i].AddTag("instance-id", s.instanceID)
```

Comment:
This technically solves the problem but could clash with a user-defined `instance-id`. Do we have any pattern around system tags (like `__var_name` in the C++ stdlib)?

Reply:
But I would recommend underscore in the middle of the tag name, as well, to avoid needing to quote it.